src/test/shared/settingsConfiguration.test.ts

```diff
@@ -84,34 +84,43 @@ describe('DefaultSettingsConfiguration', function () {
 
     describe('disablePrompt', async function () {
         let defaultSetting: any
-        const promptName = 'promptName'
-        before(async function () {
-            // This gets the default settings
+        const promptName = 'apprunnerNotifyPricing'
+        beforeEach(async function () {
+            // Force the default settings.
             await sut.writeSetting(PROMPT_SETTING_KEY, {}, vscode.ConfigurationTarget.Global)
             defaultSetting = sut.readSetting(PROMPT_SETTING_KEY)
         })
 
         const scenarios = [
             {
-                testValue: { promptName: true, other: false },
-                expected: { promptName: true, other: false },
+                testValue: { apprunnerNotifyPricing: true, other: false },
+                expected: { apprunnerNotifyPricing: true, other: false },
                 desc: 'stays suppressed',
             },
-            { testValue: { promptName: false }, expected: { promptName: true }, desc: 'suppresses prompt' },
-            { testValue: undefined, expected: undefined, desc: 'writes nothing if undefined' },
```

Comment:
Deleted this test case. @gsweetwood I don't understand why we want this behavior, or how this test passed before. `sut.disablePrompt()` is called in line 109 which necessarily sets a non-undefined value.